install/ubuntu-installation-guide.md

```diff
@@ -3,7 +3,7 @@
 > **Notes:**
 >
 > - Kata Containers packages are available for [Ubuntu\*](https://www.ubuntu.com)
->   versions **16.04** and **17.10** (currently `x86_64` only).
+>   versions **16.04**, **17.10** and **18.04** (currently `x86_64` only).
```

Comment:
@GabyCT Your commit message says 18.10, but you have added 18.04 here.

Reply:
@amshinde sorry my mistake, it is 18.04